eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -43,18 +43,13 @@ stages:
                         PackageName: ${{artifact.name}}
                         ServiceName: ${{parameters.ServiceDirectory}}
                         ForRelease: true
-                    - template: /eng/pipelines/templates/steps/stage-artifacts.yml
```

Comment:
I assume we can remove the stage-artifacts.yml file completely now.

Reply:
I am still using this in one place where document and tgz are both uploaded into docc publishing step. I am checking if we can avoid that as well.